### PR TITLE
Wave3: Adding docs for remaining pack errors and warnings

### PR DESCRIPTION
**Pull request by mishra14**
Working towards: https://github.com/NuGet/Home/issues/6498

This PR adds docs for the remaining pack errors and warnings.

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5106?branch=pr-en-us-1014) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5001.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5001.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5001?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5003.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5003.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5003?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5004?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5005.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5005.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5005?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5007.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5007.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5007?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5008.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5008.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5008?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5012.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5012.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5012?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5020.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5020.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5020?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5021.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5021.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5021?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5024.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5024.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5024?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5101.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5101.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5101?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5106.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5106.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5106?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5107.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5107.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5107?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5108.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5108.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5108?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5109.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5109.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5109?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5110.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5110.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5110?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5111.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5111.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5111?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5112.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5112.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5112?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5114.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5114.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5114?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5116.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5116.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5116?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5117.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5117.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5117?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5118.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5118.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5118?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5119.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5119.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5119?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5120.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5120.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5120?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5121.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5121.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5121?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5122.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5122.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5122?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5123.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5123.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5123?branch=pr-en-us-1014) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201808151806071027-1014%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zfdcwZqD8%2F0oI0Nx5LTuwjfbdK%2FSWLphYXIKFiVbVLg%3D&st=2018-08-15T18%3A02%3A11Z&se=2018-09-15T18%3A07%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5001.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+f1_keywords:
+  - NU5001
+---
+
+# NuGet Error NU5001
+<pre>Unable to output resolved nuspec file because it would overwrite the original at 'F:\project\project.nuspec'.</pre>
+
+### Issue
+
+The NuGet pack operation was invoked with the `-InstallPackageToOutputPath` option. But the output path already contained a  nuspec file.
```
Comment (loic-sharma) [inline docs/reference/errors-and-warnings/NU5001.md]: `option. But the output path already contained a nuspec file` -> `option but the output path already has a nuspec file`

File: docs/reference/errors-and-warnings/NU5001.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+### Solution
+
+Please ensure that the output path being passed the to NuGet pack command does not already contain a nuspec file.
```
Comment (loic-sharma) [inline docs/reference/errors-and-warnings/NU5001.md]: How about: `Please ensure that the output path passed the to NuGet pack command does not contain a nuspec file.`?

File: docs/reference/errors-and-warnings/NU5004.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ f1_keywords:
 
 ### Issue
 
-NuGet pack operation was performed using `-Symbol` option, but the folder containing the nuspec file contained no assembly files. 
+The NuGet pack operation was performed using `-Symbol` option. But the folder containing the nuspec file contained no assembly files.
```
Comment (loic-sharma) [inline docs/reference/errors-and-warnings/NU5004.md]: Why did you break this up into two sentences? I think it read better as one sentence.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5106?branch=pr-en-us-1014) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5001.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5001.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5001?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5003.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5003.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5003?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5004?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5005.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5005.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5005?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5007.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5007.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5007?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5008.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5008.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5008?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5012.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5012.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5012?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5020.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5020.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5020?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5021.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5021.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5021?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5024.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5024.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5024?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5101.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5101.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5101?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5106.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5106.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5106?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5107.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5107.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5107?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5108.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5108.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5108?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5109.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5109.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5109?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5110.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5110.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5110?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5111.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5111.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5111?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5112.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5112.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5112?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5114.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5114.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5114?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5116.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5116.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5116?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5117.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5117.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5117?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5118.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5118.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5118?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5119.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5119.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5119?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5120.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5120.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5120?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5121.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5121.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5121?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5122.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5122.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5122?branch=pr-en-us-1014) |
[docs/reference/errors-and-warnings/NU5123.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs3/docs/reference/errors-and-warnings/NU5123.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5123?branch=pr-en-us-1014) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201808162217263710-1014%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=20leSscRQwhxH%2FQ0d9TvJLN2%2FJx31xvA6csIMqMgu28%3D&st=2018-08-16T22%3A13%3A29Z&se=2018-09-16T22%3A18%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mishra14** [comment]
cc: @rohit21agrawal

**Pull request merged**